### PR TITLE
Add srcSet support for the <Img /> component.

### PR DESCRIPTION
**Pull request by dashed**
Closes https://github.com/palmerhq/the-platform/issues/42

### CODE REVIEW

**Comment by crosscompile** [comment]
Great, thanks for the PR!

File: src/Img.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import { createResource } from './createResource';
 
-export const ImgResource = createResource(src => {
+const hashingFn = ({ src, srcSet }) => `${src}${srcset}`;
```
Comment (Cryrivers) [inline src/Img.js]: Typo here:
`srcset` -> `srcSet`

Comment (Cryrivers) [reply src/Img.js]: and this is probably a reason that #39 needs to be merged asap. 😄

Comment (crosscompile) [reply src/Img.js]: Yikes good catch!  Will take a look at #39 again tomorrow morning.  We also need to set up some PR builds to catch issues like these.

Comment (dashed) [reply src/Img.js]: @Cryrivers 😬thanks for that catch!

Comment (dashed) [reply src/Img.js]: @Cryrivers @crosscompile Just in case, I created a PR to fix this typo: https://github.com/palmerhq/the-platform/pull/54

Comment (Cryrivers) [reply src/Img.js]: @crosscompile sure thing. let me update my branch

**Pull request merged**